sound/soc/intel/common/soc-acpi-intel-ptl-match.c

```diff
@@ -206,6 +215,15 @@ static const struct snd_soc_acpi_adr_device rt1320_1_group2_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device rt1320_2_group1_adr[] = {
+	{
+		.adr = 0x000230025D132001ull,
+		.num_endpoints = 1,
+		.endpoints = &spk_r_endpoint,
+		.name_prefix = "rt1320-1"
```

Comment:
not sure I follow why this one needs to have the '-1' suffix for group 1 but the rt712 above does not. IIRC the -1 and -2 where related to amps, not groups.

Reply:
The '-1' suffix will be used by the ucm. https://github.com/alsa-project/alsa-ucm-conf/blob/master/ucm2/sof-soundwire/rt1320.conf. It assume the '-1', '-2' prefix in the control names.